crates/hyperswitch_connectors/src/connectors/archipel.rs

```diff
@@ -873,17 +885,17 @@ impl ConnectorIntegration<RSync, RefundsData, RefundsResponseData> for Archipel
         req: &RefundSyncRouterData,
         connectors: &Connectors,
     ) -> CustomResult<Option<Request>, errors::ConnectorError> {
+        let auth_details = archipel::ArchipelAuthType::try_from(&req.connector_auth_type)?;
         let url = &self.get_url(req, connectors)?;
         let headers = self.get_headers(req, connectors)?;
-        let body = self.get_request_body(req, connectors)?;
 
         Ok(Some(
             RequestBuilder::new()
                 .method(Method::Get)
                 .url(url)
                 .attach_default_headers()
                 .headers(headers)
-                .set_body(body)
+                .add_ca_certificate_pem(auth_details.ca_certificate)
```

Comment:
Refund sync doesn't need a body? Is this change intentional

Reply:
Yes! @SanchithHegde refund sync does not need a body, so removed it.